cegs_portal/search/static/search/js/tables.js

```diff
@@ -18,10 +17,8 @@ function featureTable(features, regionID = "dnafeature") {
             e("th", "Parent"),
         ]),
     ]);
-    for (const feature of features) {
-        newTable.append(
-            e("tr", {class: "data-row"}, [
-                e("td", e("a", {href: `/search/feature/accession/${feature.accession_id}`}, feature.accession_id)),
+        for (const feature of features) {
+            let row = e("tr", { 'data-href': `/search/feature/accession/${feature.accession_id}`}, [
```

Comment:
the row should no longer have the class `data-row` ?

Reply:
Removed it and row hover still works. We should be ok.